journals/apps/api/v1/views.py

```diff
@@ -62,3 +64,31 @@ def _revoke_access(self, order_number):
                     access_record.journal,
                     access_record.order_number)
         return HttpResponse()
+
+
+class UserPageVisitViewSet(viewsets.ModelViewSet):
+    """API for UserPageVisit model"""
+    lookup_field = 'user_id'
+    queryset = UserPageVisit.objects.all()
+    serializer_class = UserPageVisitSerializer
+    permission_classes = (IsAdminUser,)
+
+    def retrieve(self, request, *args, **kwargs):   # pylint: disable=unused-argument,useless-suppression
```

Comment:
If we are overriding `retrieve` to filter visits by `page_id` it think we can use [SearchFilter](http://www.django-rest-framework.org/api-guide/filtering/#searchfilter) instead.

Reply:
@bill-filler I tried this.